tenable/tenableone/inventory/assets_export/api.py

```diff
@@ -0,0 +1,144 @@
+                Supported formats include: CSV and JSON. Defaults to DatasetFileFormat.JSON.
+
+        Returns:
+            ExportRequestId:
+                The export request ID.
+
+        Examples:
+            >>> export_id = tenable_one.inventory.assets_export.export(
+            ...     properties="cpe,version,file_location",
+            ...     sort_by="name",
+            ...     sort_direction=SortDirection.ASC,
+            ...     file_format=DatasetFileFormat.CSV
+            ... )
+            >>> print(export_id.export_id)
+
+        """
+        # Build query parameters
+        params = {}
+        if properties is not None:
+            params['properties'] = properties
+        if use_readable_name is not None:
+            params['use_readable_name'] = use_readable_name
+        if max_chunk_file_size is not None:
+            params['max_chunk_file_size'] = max_chunk_file_size
+        if sort_by is not None and sort_direction is not None:
+            params['sort'] = f'{sort_by}:{sort_direction.value}'
+        if file_format is not None:
+            params['file_format'] = file_format.value
+
+        # Build request body
+        payload = None
+        if filters is not None:
+            payload = DatasetExportRequest(
+                filters=[filter.model_dump(mode='json') for filter in filters]
+            ).model_dump(mode='json', exclude_none=True)
+
+        response = self._post(
+            'api/v1/t1/inventory/export/assets', json=payload, params=params
+        )
+        return ExportRequestId(**response)
+
+    def status(self, export_id: str) -> ExportRequestStatus:
+        """
+        Get export status
+
+        Args:
+            export_id (str):
+                The export ID to check status for.
+
+        Returns:
+            ExportRequestStatus:
+                The export status information.
+
+        Examples:
+            >>> status = tenable_one.assets_export.status("export-123")
+            >>> print(f"Status: {status.status}")
+            >>> print(f"Chunks available: {status.chunks_available}")
+
+        """
+        response = self._get(f'api/v1/t1/inventory/export/{export_id}/status')
+        return ExportRequestStatus(**response)
+
+    def download(self, export_id: str, chunk_id: str) -> bytes:
+        """
+        Download export
+
+        Args:
+            export_id (str):
+                The export ID.
+            chunk_id (str):
+                The chunk ID to download.
+
+        Returns:
+            bytes:
+                The exported data as bytes.
+
+        Examples:
+            >>> data = tenable_one.assets_export.download("export-123", "0")
+            >>> with open("assets_export.csv", "wb") as f:
+            ...     f.write(data)
+
+        """
+        response = self._get(
+            f'api/v1/t1/inventory/export/{export_id}/download/{chunk_id}', stream=True
+        )
+        return response.content
```

Comment:
This will load the entire response into memory.  If a file-like object is the ultimate storage goal here, should we ask for the file-like and just use it to stream the file in?